### PR TITLE
[Task #72599] Refactor Code Repository

### PR DESCRIPTION
**Pull request by DyannT**
## Related Tickets:
- [#72599](https://edu-redmine.sun-asterisk.vn/issues/72599)

## WHAT this PR does?
- Refactor Repository
- Edit Unit test
- Fix bug after unit test

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Comment by DyannT** [comment]
ready

File: app/Http/Controllers/FollowController.php [hunk 1]
```diff
@@ -2,25 +2,25 @@
 
 namespace App\Http\Controllers;
 
-use App\Repository\FollowRepository;
+use App\Repository\Interfacee\FollowRepositoryI;
```
Comment (ghost) [inline app/Http/Controllers/FollowController.php]:
```suggestion
use App\Repository\Interfacee\FollowRepositoryInterface;
```
fix tương tự các chỗ khác nha :3

**Comment by DyannT** [comment]
ready

File: app/Http/Controllers/DashboardController.php [hunk 2]
```diff
@@ -4,9 +4,9 @@
 
 use App\Http\Requests\PasswordRequest;
 use App\Http\Requests\ProfileRequest;
-use App\Repository\FollowRepository;
-use App\Repository\PostRepository;
-use App\Repository\UserRepository;
+use App\Repository\Interfacee\FollowRepositoryInterface;
```
Comment (ghost) [inline app/Http/Controllers/DashboardController.php]:
```suggestion
use App\Repository\Interface\FollowRepositoryInterface;
```

**Comment by DyannT** [comment]
ready

**Pull request merged**